### ISSUE TITLE
Hide "Free Space: ?" on start

### ISSUE THREAD

**Issue by boshevski** [Storj/farmer-gui#211]
Hide "Free Space: ?" on start, and display after Storage Location has been selected.

**Comment by littleskunk** [comment]
@boshevski the old version was displaying free space on startup without problems. It looks like a bug to me. I don't think we should hide the bug.

**Comment by FloGerlich** [comment]
I have the same problem. On V0.5 it did show it correctly, now it just shows '?'.
Running W10, NTFS formatted - over 3TB of free space.

**Comment by littleskunk** [comment]
I should be able to fix that bug myself. The reason for this bug is the name "Storj Share". The space is a little problem but I am sure I can fix that.

```
Error: Command failed: C:\WINDOWS\system32\cmd.exe /s /c "C:\Users\jensh_000\AppData\Roaming\Storj Share\diskspace.exe drive-D"
Der Befehl "C:\Users\jensh_000\AppData\Roaming\Storj" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.



    at ChildProcess.exithandler (child_process.js:225:12)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:818:16)
    at Socket.<anonymous> (internal/child_process.js:319:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```

**Comment by littleskunk** [comment]
http://stackoverflow.com/questions/16395612/unable-to-execute-child-process-exec-when-path-has-spaces

**Comment by littleskunk** [comment]
FYI: Windows escape char is ^ (Storj^ Share). Fixed with the next version 0.7.5.
